sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -74,14 +74,20 @@ class RocksDB(
     loggingId: String = "",
     useColumnFamilies: Boolean = false) extends Logging {
 
-  case class RocksDBSnapshot(checkpointDir: File, version: Long, numKeys: Long) {
+  case class RocksDBSnapshot(
+    checkpointDir: File,
+    version: Long,
+    numKeys: Long,
+    fileMappings: RocksDBFileMappings) {
```

Comment:
if we call it `capturedFileMappings` everywhere else, why not call it this name here too?

Reply:
Must have missed this, will update.